packages/codemod/src/utils/prop-utils.ts

```diff
@@ -0,0 +1,19 @@
+import { ASTPath, JSXOpeningElement } from "jscodeshift";
+
+export function updatePropName(
+  elementPath: ASTPath<JSXOpeningElement>,
+  propsNamesMappingOldToNew: Record<string, string>
+): void {
+  const attributes = elementPath.node?.attributes;
+  if (!attributes) return;
+
+  attributes.forEach(attr => {
+    if (
+      attr.type === "JSXAttribute" &&
```

Comment:
Consider adding question marks after attr and attr?.name

Reply:
`attr.type` would always exist if we're inside the `forEach` loop
`attr.name` throws TS errors if the `attr.type === "JSXAttribute"` isn't checked.
so it means that if `attr.type === "JSXAttribute"` is `true`, then `attr.name` would always exist
same for `attr.name.type` and `attr.name.name`.

As those would always exist, I think it is making the code more readable having no optional chaining so the reader would know those would always exist

<img width="1042" alt="image" src="https://github.com/mondaycom/vibe/assets/30905362/31853f67-65c8-4e96-bc97-71527acf741e">
<img width="728" alt="image" src="https://github.com/mondaycom/vibe/assets/30905362/868257e8-acac-415a-9b40-08d5fce877f8">